docs/one-wire-devices.md

```diff
@@ -0,0 +1,157 @@
+---
+title: One-Wire Devices
+---
+
+One-wire devices including but not limited to temperature loggers, battery monitors, humidity sensors, voltage and current sensors can be used with a wide range of controllers and processors in the Omega2 family of modules through One-wire communication protocol. 
```

Comment:
The Omega2 modules are all processors. Let's simplify this by saying "... sensors can be used with the Omega2 family of modules through One-wire communication protocol."

I know this is a small detail, but I want to drive home that we should be succinct in these docs and not introduce confusing terms